vadovas/katalogas.rst

```diff
@@ -1716,6 +1716,95 @@ Plano patvirtinimas organizacijos vardu
+Taip pat, generuoti sutarties dokumentus, juos peržiūrėti, atsisiųsti, pasirašyti ir įkelti.
+
+**Pastaba:** Sutartis yra sudaromos atskirai su kiekviena pasirinktų duomenų
+šaltinių organizacija. *Pavyzdžiui:* pasirinkus tris duomenų rinkinius iš tos
+pačios organizacijos bus sugeneruota viena sutartis.
+
+
+Sutarčių sąrašo peržiūra
+------------------------
+
+Sutarčių sąrašo formoje matysite visas sukurtas panaudojimo atvejo sutartis. Taip pat, kiekvienos
+sutarties duomenis teikiančią organizaciją, sutarties būseną ir paskutinį duomenų atnaujinimo laiką.
+
+| |image_panaudojimoatvejai_sutartys_1|
+| *Sutarčių sąrašo peržiūra*
+
+Sutarčių būsenų paaiškinimai:
+
+- **Pateikta** – Sutartis sukurta. Sutarties (PDF) dokumentas dar nesukurtas. Sukurti sutarties
+  dokumentą galima :ref:`sutarties peržiūros puslapyje <sutarties-perziura>`.
+  Sukūrus sutarties dokumentą, sutarties būsena taps **Suformuota**.
+- **Suformuota** – Sutarties (PDF) dokumentas sukurtas, bet nepasirašytas. Pasirašyti
+  sutarties dokumentą galima :ref:`sutarties peržiūros puslapyje <sutarties-perziura>`.
+  Pasirašius - sutarties būsena taps **Inicijuota**.
+- **Inicijuota** – Sutarties dokumentas pasirašytas duomenų gavėjo. Kai sutartį pasirašys
+  duomenų tiekėjas - sutarties būsena taps **Pasirašyta**.
+- **Pasirašyta** – Sutarties dokumentas pasirašytas duomenų gavėjo ir duomenų tiekėjo. Kai sutarties
+  duomenys bus sėkmingai sinchronizuoti su duomenų agentu, sutarties būsena taps **Aktyvi**.
+- **Aktyvi** – Sutartis sėkmingai sinchronizuota su agentu. Galima gauti duomenis.
+- **Nutraukta** – Sutartis nutraukta. Duomenų gauti nebegalima.
+
+
+Sutarties generavimas
+---------------------
+
+Generuoti sutartis (sutarties įrašus, kuriems vėliau bus galima generuoti sutarties
+dokumentus) galima paspaudus mygtuką "Generuoti sutartis" dešiniajame viršutiniame kampe.
+
+Sutartys bus sukurtos kiekvienai organizacijai atskirai. Atsidariusioje formoje bus galima
+pasirinkti norimus leidimus iš kiekvieno organizacijos duomenų rinkinio.
+
+.. _sutarties-perziura:
+
+Detali sutarties peržiūra
+-------------------------
+
+Peržiūrėti detalesnę sutarties informaciją galima sutarties peržūros puslapyje. Į jį patekti galima
+spaudžiant "Peržiūrėti" sutarčių sąrašo formoje, norimos sutarties eilutėje.
+
+Sutarties peržiūros puslapyje galima matyti platesnę informaciją apie sutartį: sutarties identifikatorių,
+panaudojimo atvejį, teikėjo ir gavėjo organizacijas, būseną, pasirinktus leidimus, paskutinę
+sinchronizacijos datą ir visus įkeltus ar sugeneruotus sutarties dokumentus. Visus sutarties dokumentus
+taip pat galima ir parsisiųsti.
+
+| |image_panaudojimoatvejai_sutartys_2|
+| *Detaili sutarties peržiūra*
+
+Sutarties dokumentai gali būti:
+
+- Sutarties generavime naudojamų duomenų ODRL JSON failas.
+- Sugeneruotas sutarties PDF failas.
+- Duomenų gavėjo pasirašytas sutarties ADOC failas.
+- Duomenų gavėjo ir duomenų teikėjo pasirašytas sutarties ADOC failas.
```

Comment:
I'd format all of these in some way (italic or bold or something else):
- PDF
- ADOC
- ODRL JSON